deploy/kubernetes/metricbeat-kubernetes.yaml

```diff
@@ -14,12 +14,47 @@ data:
       # Reload module configs as they change:
       reload.enabled: false
 
-    # To enable hints based autodiscover uncomment this:
-    #metricbeat.autodiscover:
-    #  providers:
-    #    - type: kubernetes
-    #      node: ${NODE_NAME}
-    #      hints.enabled: true
+    metricbeat.autodiscover:
+      providers:
+        # To enable hints based autodiscover uncomment this:
+        #- type: kubernetes
+        #  node: ${NODE_NAME}
+        #  hints.enabled: true
+        # Uncomment the following to enable leader election provider that handles
+        # singleton instance configuration across the Daemonset Pods of the whole cluster
+        # in order to monitor some unique data sources, like kube-state-metrics.
+        # When enabling this remember to also delete the Deployment or just set the replicas of the
+        # Deployment to 0.
+        #- type: kubernetes
+        #  scope: cluster
+        #  node: ${NODE_NAME}
+        #  unique: true
+        #  # identifier: <lease_lock_name>
```

Comment:
This might be confusing: under which circumstances should I uncomment? What the value should be in that case? 
I'd remove this line altogether.

Reply:
👍 I will remove it